.github/workflows/containers.yml

```diff
@@ -30,3 +30,15 @@ jobs:
 
       - name: Push App Run container
         run: docker push ccfciteam/ccf-app-run:${{steps.tref.outputs.tag}}
+
+      - name: Tag latest CI container
+        run: docker tag ccfciteam/ccf-app-ci:${{steps.tref.outputs.tag}} ccfciteam/ccf-app-ci:latest
```

Comment:
We can add this tag when we `docker build`, with multiple `-t` args. That seems less error-prone that repeating the original tag name construction here.

Reply:
Ah great, I didn't know you could -t more than once.